app/src/main/java/com/example/mob_lab1/HomeActivity.java

```diff
@@ -0,0 +1,33 @@
+package com.example.mob_lab1;
+
+import androidx.appcompat.app.AppCompatActivity;
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import com.google.firebase.auth.FirebaseAuth;
+
+public class HomeActivity extends AppCompatActivity {
+    Button btnLogout;
+    FirebaseAuth mFirebaseAuth;
+    private FirebaseAuth.AuthStateListener mAuthStateListener;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_home);
+        btnLogout = findViewById(R.id.logout);
+
+        btnLogout.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                FirebaseAuth.getInstance().signOut();
+                Intent intToMain = new Intent(HomeActivity.this, MainActivity.class);

```

Comment:
intToMain - некоректна назва
```suggestion
                Intent intentToMain = new Intent(HomeActivity.this, MainActivity.class);
```